MAINTAINERS

```diff
@@ -31,6 +31,78 @@ to appreciate the absence of bugs, the slow but steady improvement in stability,
 or the reliability of a release process. But those things distinguish a good
 project from a great one.
 """
+    [Rules.adding-maintainers]
+
+        title = "How are maintainers added?"
+
+        text = """
+Maintainers are first and foremost contributors that have shown they are
+committed to the long term success of a project. Contributors wanting to
+become maintainers are expected to be deeply involved in contributing code,
+pull request review, and triage of issues in the project for more than three
```

Comment:
The "for more than three months" is to differentiate contributors that have a short "streak" of contributions in the repository, but stop after that. When selecting maintainers, you'd be looking for people that you can rely on to carry the project, hence looking activity over a longer period.